src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -395,6 +413,12 @@ public virtual string CustomMethod
 
         internal virtual void ValidateParameters()
         {
+            if (MyInvocation.BoundParameters.ContainsKey(nameof(Path)) && MyInvocation.BoundParameters.ContainsKey(nameof(LiteralPath)))
```

Comment:
could you add some commentary why you didn't use parameter sets here? If we ever change our parameter set logic, this could be one of those places where we could take advantage of it.

Reply:
We have already 3 parameter sets and class inheritance. I'm afraid adding two more parameter sets highly complicates the code and may turn into a headache. I'd prefer to postpone this to another PR if we really want this.

Reply:
just to be clear, I wasn't suggesting additional parameter sets. Your reasoning is sound, it's pretty complicated, and doesn't need to be more so. I was just looking for commentary in the code as to _why_ so others doing reviews or later maintenance would understand.

Reply:
The change was reverted as PowerShell Committee request.